src/memory_server.ml

```diff
@@ -278,22 +299,51 @@ let calculate_boot_time_host_free_memory () =
       pages_to_kib (Int64.of_nativeint boot_time_host_free_pages) in
     Int64.mul 1024L boot_time_host_free_kib
 
+let calc_constant_boot_time_host_free_memory constant_count interval =
+  debug "Check boot time host free memory: constant-count=%d check-interval=%f(seconds)" constant_count interval;
+  let rec calc_constant existing_results =
+    let free_memory = calculate_boot_time_host_free_memory () in
+    debug "Calculating boot time host free memory: %Ld" free_memory;
+    if (List.length existing_results) = (constant_count - 1) then
+      if Listext.List.filter_map (fun x -> if x <> free_memory then Some(true) else None) existing_results = [] then begin
```

Comment:
Remove `Listext.` and the import added to use it. Ocaml's Stdlib now has `List.filter_map` as well.

Reply:
Correct! I added `Listext` because the first time building this binary I was using the old version libaray.